server/index.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const createList = require('../db/index.js').createList;
+const addPhotos = require('../db/index.js').addPhotos;
+const grabPhotosFromOne = require('../db/index.js').grabPhotosFromOne;
+const grabPhotosFromAll = require('../db/index.js').grabPhotosFromAll;
+
+
+
+const app = express();
+
+app.use(express.static(__dirname + '/../dist'));
+
+const PORT = 3003;
+app.use(bodyParser.json());
+
+app.post('/api/photo-gallery-list-add', (req, res) => { //Used to populate page, or, in a real scenario, creating a page's initial batch of photos
+  createList(req.body, (err, result) => {
+    if (err) {
+      res.sendStatus(500);
+    } else {
+      res.json(result);
+    }
+  });
+});
+
+app.post('/api/photo-gallery-list-add/:restId', (req, res) => { //Probably won't be implemented on FDC, stretch goal
```

Comment:
Awesome that you write the skeleton for remembering what you thought about but know that you're time constrained. Very smart way of being able to come back to it instead of forgetting in case you do move ahead of schedule!